build.gradle

```diff
@@ -18,23 +18,31 @@ subprojects {
 
     ext {
         autoValueVersion = '1.7.4'
+        slf4jVersion = '1.7.30'
         googleCloudVersion = '1.0.2'
+        cloudMonitoringVersion = '2.0.1'
         openTelemetryVersion = '0.7.0'
         junitVersion = '4.13'
+        mockitoVersion = '3.5.10'
         opentelemetryOperationsVersion = "0.7.1"  // CURRENT_RELEASE_VERSION
 
         libraries = [
                 auto_value_annotations       : "com.google.auto.value:auto-value-annotations:${autoValueVersion}",
                 auto_value                   : "com.google.auto.value:auto-value:${autoValueVersion}",
                 google_cloud_core            : "com.google.cloud:google-cloud-core:${googleCloudVersion}",
-                google_cloud_grpc            : "com.google.api.grpc:grpc-google-cloud-trace-v2:${googleCloudVersion}",
                 google_cloud_trace           : "com.google.cloud:google-cloud-trace:${googleCloudVersion}",
+                google_cloud_trace_grpc      : "com.google.api.grpc:grpc-google-cloud-trace-v2:${googleCloudVersion}",
+                google_cloud_monitoring      : "com.google.cloud:google-cloud-monitoring:${cloudMonitoringVersion}",
+                google_cloud_monitoring_grpc : "com.google.cloud:grpc-google-cloud-monitoring-v3:${cloudMonitoringVersion}",
+                slf4j                        : "org.slf4j:slf4j-api:${slf4jVersion}",
                 opentelemetry_api            : "io.opentelemetry:opentelemetry-api:${openTelemetryVersion}",
                 opentelemetry_sdk            : "io.opentelemetry:opentelemetry-sdk:${openTelemetryVersion}",
                 opentelemetry_operations_java: "com.google.cloud.opentelemetry:exporter-trace:${opentelemetryOperationsVersion}"
         ]
         testLibraries = [
-                junit: "junit:junit:${junitVersion}",
+                junit       : "junit:junit:${junitVersion}",
+                mockito     : "org.mockito:mockito-inline:${mockitoVersion}",
```

Comment:
I can't find any usage of Mockito in the code? Are you sure you need this?

Reply:
I meant to add this change in the next test PR but accidentally added it to this one instead, I've removed it here now